.github/workflows/cuckoojs-schematics-npm-publish.yml

```diff
@@ -29,13 +43,28 @@ jobs:
     - name: Build artifact
       run: npm run build -w @guidesmiths/cuckoojs-schematics
     - name: Publish package to npm
+      if: ${{ needs.check-cuckoojs-schematics-version.outputs.version-changed }} === "true"
```

Comment:
This new approach forces us to run all the job even if we don't need it. But I don't know of a better way right now. So let's go for it.

Reply:
Yep, I know. I didn't figure out how to do it differently. Let's try this approach, at least to unblock the release process. I think we should explore other options for managing the dependencies and their releases, like `Lerna`